src/session-status-hooks.ts

```
@@ -68,24 +69,32 @@ mv -f "$TMPFILE" "$STATUS_DIR/$SESSION_ID.json"
+const getSettingsPaths = (): string[] =>
+  getScannableAccounts().map((a) => path.join(a.dir, 'settings.json'));
 
-  // Create status directory
-  fs.mkdirSync(STATUS_DIR, { recursive: true });
+/** Back up a settings.json before we modify it. */
+const backupSettings = (settingsPath: string): void => {
+  try {
+    if (fs.existsSync(settingsPath)) {
+      fs.copyFileSync(settingsPath, `${settingsPath}.codev-bak`);
+    }
+  } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)